README.md

```diff
@@ -39,19 +55,36 @@ You can now build the PureScript source code with:
 npm run build
 ```
 
-You can produce a bundled JS file you can run in the browser with:
+### Launching the App
+
+You can launch your app in the browser with:
 
 ```sh
-# An alias for `spago bundle-app --to dist/app.js`
-npm run bundle
+# An alias for `parcel dev/index.html --out-dir dev-dist --open`
+npm run serve
 ```
 
-This deposits a bundled JS file named `app.js` in the `dist` directory. You can view your running Halogen app by opening the `dist/index.html` file.
+### Development Cycle
+
+If you're using an [editor](https://github.com/purescript/documentation/blob/master/ecosystem/Editor-and-tool-support.md#editors) that supports [`purs ide`](https://github.com/purescript/purescript/tree/master/psc-ide) or are running [`pscid`](https://github.com/kRITZCREEK/pscid), you simply need to keep the previous `npm run serve` command running in a terminal. Any save to a file will trigger an incremental recompilation, rebundle, and web page refresh, so you can immediately see your changes.
+
+If your workflow does not support automatic recompilation, or if you add, remove, or modify module names, then you will need to manually re-run `npm run build`.
+
+:warning: `purs ide` only rebuilds one module at a time, so sometimes the bundle will end up in an inconsistent state, resulting in runtime errors. This occurs when a change is made in one module that breaks other modules that depend on it. The solution is to run a full build (`npm run build`) when a change like this is made, as the compiler will force you to resolve those errors.
+
+### Production
+
+When you are ready to create a minified bundle for deployment, run the following command:
+```sh
+npm run build-prod
+```
 
-Alternatively, if you use an editor that supports `purs ide` or if you are running [`pscid`](https://github.com/kRITZCREEK/pscid), then you can get near-instant builds of the app while you work:
+Parcel output appears in the `./dist/` directory.
 
+You can test the production output locally with a tool like [`http-server`](https://github.com/http-party/http-server#installation). It seems that `parcel` should also be able to accomplish this, but it unfortunately will only serve development builds locally.
```

Comment:
This is a bit disappointing. It's a nice part of Webpack: it comes with a dev server out of the box.

Reply:
Parcel also includes a "dev server", but they're really strict about the "dev" part because it only works for dev builds. Hoping they'll eventually enable a way for this to work with production builds too.